### DISCUSSION TITLE
Deploying to a VM and setting up a domain

### DISCUSSION THREAD

**Discussion by josefmtd** [developmentseed/titiler#334] [Q&A] [answered]
Hello, I tried using the Docker image to build a TiTiler application on my VM, but I wasn't able to use e.g. the COG viewer /cog/viewer since the request still uses "http://localhost:8000" instead of a domain I assigned, is there a way to change this while building the Docker container? Thanks!

**Comment by vincentsarago** [comment, marked as answer]
you can have a look at https://github.com/tiangolo/uvicorn-gunicorn-docker#advanced-usage to see the different options you can set

**Reply by josefmtd** [reply]
![image](https://user-images.githubusercontent.com/28034179/122931498-263d3500-d397-11eb-8018-22399b42d991.png)
Thanks for the very quick reply! This is the response I get, I will look at uvicorn, to find the settings I need to go through.

**Reply by josefmtd** [reply]
D'oh, I messed up my NGINX! It works just fine now, thanks for the help.
